### DISCUSSION TITLE
Welcome to iot-inspector-client Discussions!

### DISCUSSION THREAD

**Discussion by crazyideas21** [noise-lab/iot-inspector-client#102] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by semloh99** [comment]
I have just installed on a Windows 10 PC. All of my network devices show up in the Dashboard except for the machine that IoT Inspector is installed on. Is this expected behavior?

**Reply by semloh99** [reply]
I also noticed an unknown device in the activity on the Overview page. It was not shown on the device list. I was able to bring it up in the Device Details by double clicking on the entry in "Who are my devices talking to?" on the Overview. It turns out to be my DSL Modem/Router and network gateway and I updated the device name.
It now has the correct device name in Activity but does not show up in the Device List.

**Reply by crazyideas21** [reply]
> Is this expected behavior?

Yes. By design.

**Reply by crazyideas21** [reply]
> does not show up in the Device List.

By default, we hide the gateway and the Inspector-running computer from the device list.

It is by design that the IoT Inspector host and the gateway are NOT in the list (see [source code](https://github.com/nyu-mlab/iot-inspector-client/blob/master/core/packet_processor.py#L55)) — which makes it easy to process the ARP-spoofed traffic.

Do you have a particular use case to see the traffic for either devices?

**Reply by semloh99** [reply]
No but as a former QA Engineer, anomalies always bring up a question. Thanks for the quick response and the tool. I found that the device yakking the most was one I don't even use and had trouble identifying.  Chromecast attached to a TV I don't use.

**Reply by semloh99** [reply]
Not really competent to look at the code.

**Reply by crazyideas21** [reply]
> had trouble identifying

How did you identify/discover the Chromecast eventually?

We're trying to redesign device identification. How can we improve IoT Inspector to help you identify/discover devices?

**Comment by semloh99** [comment]
It was talking to Google a lot so I assumed it was some type of Google device so I walked around and found it. I unplugged it from the power and it disappeared from my connected devices.
Not very helpful, I'm afraid. My router didn't know what it was either.

⁣Get BlueMail for Android ​

**Reply by crazyideas21** [reply]
This is actually very helpful @semloh99! Many of our users have trouble identifying devices on their network. We will improve the UI and especially UX to guide our users to identify their devices (e.g., unplugging certain devices).
